partners/scientific-communities.md

```diff
@@ -9,26 +9,292 @@ JOSS <joss>
+
+1. We will create a community page in this peer review guide where you can define your guidelines for a package to become affiliated with your community. Check out the
+   [pangeo example](pangeo) as an example of what this looks like.
+2. A dedicated community page on the pyOpenSci website. Here, we will
+   list your community along with all affiliated packages that have
+   successfully undergone our review process. You can add descriptive text and graphics and branding to the page as you wish.
+
+### Packaging Support
+
+Our vibrant community is filled with experts in the Python packaging space. A
+partnership with us means that you will always have support if you run into
+Python packaging issues.
+
+### Future work
+
+We are currently working on several items to improve our review process and followup activities with package maintainers including:.
+
+- Collecting metrics and will be creating small dashboards that track package health for all accepted packages in our ecosystem.
+- Working on a bot adapted from the JOSS bot that will streamline the editorial process for peer review.
+
+## Partnership benefits to the broader scientific community
+
+The benefits for the broader scientific Python community include:
+
+- We have broad reach across the entire scientific Python ecosystem which allows us to promote your community and associated packages.
+- We have a broad view across all domains within the scientific Python ecosystem which will help us to identify overlap of package functionality across domains and potentially aggregate efforts in the future as it makes sense (i.e., if a maintainer needs to step down).
+- We offer an established peer review process will help enforce standardization of scientific Python packaging and best practices across the ecosystem. Our volunteer-lead peer review process has been operating since 2019.
+
+## How peer review works for partners
+
+The pyOpenSci peer review process can be broken down into 5 steps when a package is submitted for review:
+
+1. To kick things off, an Editor-in-Chief checks that the package passes our [bare-minimum checks](https://www.pyopensci.org/software-peer-review/how-to/editor-in-chief-guide.html#editor-checklist-template).
+2. Once Editor-in-Chief checks are passed, they assign a domain-expert editor to the review.
+
+Through our partnership, we have two options when selecting an editor.
+
+- If the editor is someone from your partner community who has authority, they can determine whether the package meets the decided-upon criteria to become affiliated with your community.
+
+* If we have a domain specific editor who is not authorized by your community to make this decision, we can ask someone from your community to make the decision around affiliated status. Once the decision is made, the review can proceed as normal.
+
+3. Once affiliated status has been determined, the editor finds 2 reviewers to begin the review.
```

Comment:
There's some inconsistent enumeration here

Reply:
ok i've fixed this locally and will push it shortly! thanks @grst !!